graph/src/data/query/error.rs

```diff
@@ -57,6 +57,8 @@ pub enum QueryExecutionError {
     UndefinedFragment(String),
     // Using slow and prefetch query resolution yield different results
     IncorrectPrefetchResult { slow: q::Value, prefetch: q::Value },
+    Panic(String),
+    EventStreamError,
```

Comment:
I wonder if these aren't too much of an implementation detail; then again... why not! They might help us debug things in the future.